cypress/e2e/shared/legends.test.ts

```diff
@@ -500,9 +500,13 @@ describe('Legends', () => {
         .click()
       cy.getByTestID('add-resource-dropdown--new').click()
 
-      cy.getByTestID('page-title')
+      cy.getByTestID('page').should('exist')
```

Comment:
wouldn't "page" exist on the dashboards list page too? Could we try and look for an element that's unique to the new dashboard page? Maybe like the "Add cell" button or something?

Reply:
Strangely, in the dashboards list, the data-testid is "empty-dashboards-list" even when the list is not empty

Reply:
@genehynson so i added a testID just for the dashboard page. That should make it unique.

Reply:
ah, awesome thanks!